google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -44,27 +62,63 @@
 
 /** Implementation of Cloud Spanner remote calls using Gapic libraries. */
 public class GapicSpannerRpc implements SpannerRpc {
+  
```

Comment:
The fields and constructor are only used to make it compile. #3039 implements this part; after that is merged I will resolve the conflicts here.

Reply:
SGTM. The PR is already merged.